src/Repeater/ServerRepeaterLauncher.ts

```diff
@@ -101,6 +101,10 @@ export class ServerRepeaterLauncher implements RepeaterLauncher {
 
     this.repeaterRunning = true;
 
+    setUser({
+      username: repeaterId
+    });
+
```

Comment:
Sentry is used across many CLI commands, so setting `username` in just one feels a bit odd. If the goal is to enable search, consider using tags instead:

```ts
setTag("bridge_id", repeaterId);
```